rio_tiler_pds/landsat/aws/landsat_collection2.py

```diff
@@ -67,11 +67,15 @@ class LandsatC2Reader(MultiBandReader):
 
     _scheme: str = "s3"
     _hostname: str = "usgs-landsat"
-    _prefix: str = "collection02/level-{_processingLevelNum}/standard/{sensor_name}/{acquisitionYear}/{path}/{row}/{scene}/{scene}"
+    _prefix: str = "collection02/level-{_processingLevelNum}/standard/{_sensor_name}/{acquisitionYear}/{path}/{row}/{scene}/{scene}"
 
     def __attrs_post_init__(self):
         """Fetch productInfo and get bounds."""
         self.scene_params = sceneid_parser(self.sceneid)
+
+        if self.scene_params["collectionCategory"] == "RT":
+            raise ValueError("RT scenes not available in the usgs-landsat bucket")
```

Comment:
Probably should raise a more specific error than `ValueError`.

Reply:
@kylebarron are we sure about this? can you share the docs that say that? 

If there is nothing official, I think it will be fine to let rasterio fire it's own exception about file not found. (in your case it will raise when trying to get the bounds, so maybe we should fire an error in `get_geometry`)

Reply:
I haven't found anything in the docs that say this, it's only from experience so far that I've seen no RT scenes in the bucket.

I think the main argument against just allowing the error is that the error is obscure... "`NoSuchKey`" and might not be obvious to users. Maybe the best solution is to print a warning? But that might disappear above a long traceback

Reply:
So maybe catch the `NoSuchKey` error and raise a more user friendly error?